src/main/java/com/epam/ta/reportportal/dao/util/TimeCalculatorUtil.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.ta.reportportal.dao.util;
+
+import java.sql.Timestamp;
+import java.time.Duration;
+import java.time.Instant;
+
+/**
+ * @author <a href="mailto:ivan_budayeu@epam.com">Ivan Budayeu</a>
+ */
+public final class TimeCalculatorUtil {
+
+	private TimeCalculatorUtil() {
+		//static only
+	}
+
+	public static Timestamp getTimeBeforeNowWithPeriod(Duration period) {
```

Comment:
probably it's better to give a more clear name for the method. some kind of:
getTimestampBackFromNow(Duration period)

Reply:
done